README.md

```diff
@@ -2,6 +2,51 @@
 
 A zoo for models tuned for OpenCV DNN with benchmarks on different platforms.
 
+Guidelines:
+- To clone this repo, please install [git-lfs](https://git-lfs.github.com/), run `git lfs install` and use `git lfs clone https://github.com/opencv/opencv_zoo`.
+- To run benchmark on your hardware settings, please refer to [benchmark/README](./benchmark/README.md).
+
+## Models & Benchmarks
+- [Face Detection](#face-detection)
+- [Text Detection](#text-detection)
+- [Text Recognition](#text-recognition)
+
+Platforms:
+- `CPU x86_64`: INTEL CPU i7-5930K @ 3.50GHz, 6 cores, 12 threads.
+- `CPU ARM`: Raspberry 4B, BCM2711B0 @ 1.5GHz (Cortex A-72), 4 cores, 4 threads.
+- `GPU CUDA`: NVIDIA Jetson Nano B01, 128-core Maxwell, Quad-core ARM A57 @ 1.43 GHz.
+
+
+### Face Detection
+
+- Input size: width = 160, height = 120.
+- Accuracy: Results on the [WIDER Face dataset](http://shuoyang1213.me/WIDERFACE/) are collected from the source of each model.
+
+| Model | Accuracy | CPU x86_64 (ms) | CPU ARM (ms) | GPU CUDA (ms) |
+|-------|----------|-----------------|--------------|---------------|
+| [YuNet](./models/face_detection_yunet)  | E: 83.4, M: 82.4, H: 70.8 | 2.17 | TBD | TBD |
```

Comment:
A more detailed description of the test method is needed. For example, whether to test multiple times to eliminate accidental effects.